### PR TITLE
First pass at Flow Arts Hut

### PR DESCRIPTION
**Pull request by ericrius1**
This is a first draft at a "flow arts hut." This is a special hut where people can go and play with toys typically associated within the flow arts community, such as rave sticks, poi, and lightsabers. Currently there are three flow art toys- a particle ball, a ravestick, and a lightsaber. These toys are entities with entity scripts that can be taken out of the rave hut and placed anywhere it is desired. More toys to come.

This PR introduces some changes to the polyline entity and small tweaks to particle effect entity

### CODE REVIEW

**Comment by ericrius1** [comment]
To Test: https://rawgit.com/ericrius1/hifi/polylineOptimizations/examples/flowArts/flowArtsHutSpawner.js

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5b52b5e5e7acfeb6bde2a726ac7e5f860b7b4b30 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6f82d680b9cb7873f091614fde07c6fd055a0ccd available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 92f96cbb2f403a556a62bae17326def1e90c4c8f available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3e6ae7c5710e99f2508022bb80082b4fcc73911b available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: aa1f77ab1c6e0294983e2f8f4c18bfa7dbef2772 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 17af34a3be36fdc30ae832f602d5f260358e5cb4 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 77264b3f99283694ef126d859680760a10e39032 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2dbb3bd84abf25e2d5d6f0fc0f90ae3d02682012 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2c02e7ad631ddec997bbcca99a462b701554d92b available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: fffe6dbb9bd32ec6dcb7ce71a8c787dcbc00df66 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1beec0dbe7c46327bc0553734fa5871bbf3fc90d available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

File: examples/flowArts/arcBall/arcBall.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    }];
+
+
+    var containerBall = Entities.addEntity({
+        type: "Sphere",
+        name: "Arc Ball",
+        script: scriptURL,
+        position: Vec3.sum(spawnPosition, {
+            x: 0,
+            y: .5,
```
Comment (imgntn) [inline examples/flowArts/arcBall/arcBall.js]: initial zeros, please! :)

File: examples/flowArts/arcBall/arcBall.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        azimuthStart: -Math.PI,
+        azimuthFinish: Math.PI,
+        emitAcceleration: {
+            x: 0,
+            y: 0,
+            z: 0
+        },
+        accelerationSpread: {
+            x: .00,
+            y: .00,
```
Comment (imgntn) [inline examples/flowArts/arcBall/arcBall.js]: initial zeros

File: examples/flowArts/arcBall/arcBallEntityScript.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                polarFinish: .0,
+                azimuthStart: .1,
+                azimuthFinish: .01,
+                emitAcceleration: {
+                    x: 0,
+                    y: 0,
+                    z: 0
+                },
+                accelerationSpread: {
+                    x: .00,
```
Comment (imgntn) [inline examples/flowArts/arcBall/arcBallEntityScript.js]: initial zeroes

File: examples/flowArts/flowArtsHutSpawner.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+//  This script creates a special flow arts hut with a bunch of flow art toys people can go in and play with
+//
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+
+Script.include("../../libraries/utils.js");
+Script.include("lightBall/LightBall.js");
```
Comment (imgntn) [inline examples/flowArts/flowArtsHutSpawner.js]: i'm not exactly sure that we have a standard, sometimes i forget myself and do random things, but usually filenames in camel case?  i.e. raveStick.js?  unless youre going for some pattern i'm not aware of (sometimes people initial case classes, are you treating entity scripts like classes?)

File: examples/flowArts/flowArtsHutSpawner.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+// RAVE ITEMS
+// var lightBall = new LightBall(basePosition);
+
+var arcBall = new ArcBall(basePosition);
+var arcBall2 = new ArcBall(Vec3.sum(basePosition, {x: -1, y: 0, z: 0}));
+var raveStick = new RaveStick(Vec3.sum(basePosition, {x: 1, y: 0.5, z: 1}));
+var lightSaber = new LightSaber(Vec3.sum(basePosition, {x: 3, y: 0.5, z: 1}));
+
+
+var modelURL = "https://s3.amazonaws.com/hifi-public/eric/models/rave/RaveRoom.fbx";
```
Comment (imgntn) [inline examples/flowArts/flowArtsHutSpawner.js]: does it assets are  in hifi-public and not hifi-content?  don't want to open the hifi-production can of worms but i'm not sure whether we're supposed to add new things to public anymore..?

File: examples/flowArts/arcBall/arcBallEntityScript.js [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+    var ArcBall = function() {
+        _this = this;
+    };
+
+    ArcBall.prototype = {
+        isGrabbed: false,
+        startNearGrab: function() {
+            //Search for nearby balls and create an arc to it if one is found
+            var position = Entities.getEntityProperties(this.entityID, "position").position
+            var entities = Entities.findEntities(position, 10);
```
Comment (imgntn) [inline examples/flowArts/arcBall/arcBallEntityScript.js]: magic number 10

File: examples/flowArts/lightBall/lightBall.js [hunk 7]
```diff
@@ -0,0 +1,145 @@
+//
+//  LightBall.js
+//  examples/lightBall
+//
+//  Created by Eric Levin on 12/17/15.
+//  Copyright 2014 High Fidelity, Inc.
```
Comment (imgntn) [inline examples/flowArts/lightBall/lightBall.js]: wrong year

File: examples/flowArts/flowArtsHutSpawner.js [hunk 8]
```diff
@@ -0,0 +1,91 @@
+//
+//  flowArtsHutSpawner.js
+//  examples/flowArts
+//
+//  Created by Eric Levin on 12/17/15.
+//  Copyright 2014 High Fidelity, Inc.
```
Comment (imgntn) [inline examples/flowArts/flowArtsHutSpawner.js]: wrong year

File: examples/flowArts/arcBall/arcBall.js [hunk 9]
```diff
@@ -0,0 +1,144 @@
+//
+//  arcBall.js
+//  examples/arcBall
+//
+//  Created by Eric Levin on 12/17/15.
+//  Copyright 2014 High Fidelity, Inc.
```
Comment (imgntn) [inline examples/flowArts/arcBall/arcBall.js]: wrong year

File: examples/flowArts/lightSaber/lightSaberEntityScript.js [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+        unload: function() {
+            Entities.deleteEntity(this.beam);
+        },
+
+        createBeam: function() {
+
+
+            this.props = Entities.getEntityProperties(this.entityID, ["position", "rotation"]);
+            var forwardVec = Quat.getFront(Quat.multiply(this.props.rotation, Quat.fromPitchYawRollDegrees(-90, 0, 0)));
+            // forwardVec = Vec3.normalize(forwardVec);
```
Comment (imgntn) [inline examples/flowArts/lightSaber/lightSaberEntityScript.js]: cleanup if unused

File: examples/flowArts/lightSaber/lightSaberEntityScript.js [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+                emitOrientation: forwardQuat,
+                emitSpeed: .4,
+                speedSpread: 0.0,
+                emitDimensions: {
+                    x: 0,
+                    y: 0,
+                    z: 0
+                },
+                polarStart: 0,
+                polarFinish: .0,
```
Comment (imgntn) [inline examples/flowArts/lightSaber/lightSaberEntityScript.js]: initial zeros

File: examples/flowArts/lightTrails.js [hunk 12]
```diff
@@ -0,0 +1,191 @@
+//
+//  lightTrails.js
+//  examples
+//
+//  Created by Eric Levin on 5/14/15.
+//  Copyright 2014 High Fidelity, Inc.
```
Comment (imgntn) [inline examples/flowArts/lightTrails.js]: wrong year

File: examples/flowArts/lightTrails.js [hunk 13]
```diff
@@ -0,0 +1,191 @@
+//
+//  lightTrails.js
+//  examples
+//
```
Comment (imgntn) [inline examples/flowArts/lightTrails.js]: description?

File: examples/flowArts/lightTrails.js [hunk 14]
```diff
@@ -0,0 +1,191 @@
...
+    return v.x === 0 && v.y === 0 && v.z === 0;
+}
+
+
+var rightController = new controller(RIGHT, Controller.findAction("RIGHT_HAND_CLICK"));
+var leftController = new controller(LEFT, Controller.findAction("LEFT_HAND_CLICK"));
+Script.update.connect(update);
+Script.scriptEnding.connect(scriptEnding);
+
+function map(value, min1, max1, min2, max2) {
```
Comment (imgntn) [inline examples/flowArts/lightTrails.js]: you already include utils, isn't this in there?

File: examples/flowArts/raveStick/RaveStick.js [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+            red: 200,
+            green: 200,
+            blue: 255
+        },
+        colorFinish: color,
+        maxParticles: 100000,
+        lifespan: 1,
+        emitRate: 1000,
+        emitOrientation: forwardQuat,
+        emitSpeed: .2,
```
Comment (imgntn) [inline examples/flowArts/raveStick/RaveStick.js]: initial zeroes

File: examples/flowArts/raveStick/raveStickEntityScript.js [hunk 16]
```diff
@@ -0,0 +1,142 @@
...
+//
+//  This entity script create light trails on a given object as it moves.
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+(function() {
+    Script.include("../../libraries/utils.js");
+    var _this;
+    // this is the "constructor" for the entity as a JS object we don't do much here
```
Comment (imgntn) [inline examples/flowArts/raveStick/raveStickEntityScript.js]: this comment doesn't make sense here.  if it were in the RaveStick function maybe.  but actually you do some stuff.  just delete it :)

File: examples/flowArts/raveStick/raveStickEntityScript.js [hunk 17]
```diff
@@ -0,0 +1,142 @@
...
+            color: {
+                red: 255,
+                green: 255,
+                blue: 255
+            },
+            textures: texture,
+            lifetime: LIFETIME
+        });
+
+
```
Comment (imgntn) [inline examples/flowArts/raveStick/raveStickEntityScript.js]: extra line

File: examples/flowArts/raveStick/raveStickEntityScript.js [hunk 18]
```diff
@@ -0,0 +1,142 @@
...
+            this.setupEraseInterval();
+        },
+
+        continueNearGrab: function() {
+            var props = Entities.getEntityProperties(this.entityID, ["position", "rotation"]);
+            var forwardVec = Quat.getFront(Quat.multiply(props.rotation, Quat.fromPitchYawRollDegrees(-90, 0, 0)));
+            forwardVec = Vec3.normalize(forwardVec);
+            var forwardQuat = orientationOf(forwardVec);
+            var position = Vec3.sum(props.position, Vec3.multiply(Quat.getFront(props.rotation), 0.2));
+            // position.z += 0.1;
```
Comment (imgntn) [inline examples/flowArts/raveStick/raveStickEntityScript.js]: do you use these?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 45ba7a3d35c12e55207acbee615757f022e655e1 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

File: examples/flowArts/raveStick/raveStickEntityScript.js [hunk 19]
```diff
@@ -0,0 +1,142 @@
...
+            Entities.deleteEntity(this.beam);
+            Entities.deleteEntity(this.trail);
+            if (this.trailEraseInterval) {
+                Script.clearInterval(this.trailEraseInterval);
+            }
+        }
+    };
+    return new RaveStick();
+
+    function computeNormal(p1, p2) {
```
Comment (imgntn) [inline examples/flowArts/raveStick/raveStickEntityScript.js]: does this actually work, after a return statement?

File: unpublishedScripts/hiddenEntityReset.js [hunk 20]
```diff
@@ -160,6 +168,111 @@
...
+                    red: 200,
+                    green: 200,
+                    blue: 255
+                },
+                colorFinish: color,
+                maxParticles: 100000,
+                lifespan: 1,
+                emitRate: 1000,
+                emitOrientation: forwardQuat,
+                emitSpeed: .2,
```
Comment (imgntn) [inline unpublishedScripts/hiddenEntityReset.js]: zeroes

File: unpublishedScripts/masterReset.js [hunk 21]
```diff
@@ -137,6 +147,110 @@ MasterReset = function() {
...
+            },
+            colorFinish: color,
+            maxParticles: 100000,
+            lifespan: 1,
+            emitRate: 1000,
+            emitOrientation: forwardQuat,
+            emitSpeed: .2,
+            speedSpread: 0.0,
+            polarStart: 0,
+            polarFinish: .0,
```
Comment (imgntn) [inline unpublishedScripts/masterReset.js]: zeroes

**Comment by hifi-gustavo** [comment]
Build console output for commit: 48ab99c34864f5ecd2734900e4d89b29f964288d available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3c66e3826830a2f9bc84697da5e4d7b61dc77423 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ba3633710a9dfb0663f386a8f532fd5bd83e1c5f available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b89e25f8f7f6fe1138c9521c9b309111f164838d available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4b4fe96b64808321652b4ca0e2a8d9ea056d5331 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4110630d1f074fdf4c710f804d10d14539424a00 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: cabc47e7a9914e9e2985e932de6f74310d3bb71f available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 876d8ab6b25a427ae16acbe459ea3e4f01b05348 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f19d1a3067ccf7e586c17ac59d8e45092db1a907 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8d80c06f02eedad5c5d7a9395152a87d902d3739 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ba993eccd4686e5c60b8e636dd3e7b378ee232a7 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1031489ada2b4053f388823f923d531c0de65e25 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by RyanDowne** [comment]
Does this whole thing need to be merged just to get a light stick in toybox? Maybe you could create a sep PR that just adds it to toybox?

**Comment by imgntn** [comment]
i'm about to merge this once eric fixes particle system alignment on the lightsaber.  it's pretty much good to go.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 653e3ecdcc0cec85d61cf22b26ac7f6d1efc7780 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e8803ad9aecd6c66620ecd1de1436887f870988d available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: bb28369dee44af6f9eeff80a359b47cb6b7f5d87 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6fc98bd5773d7aa93c9b29692665e59a9c577a36 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 51630734dfac1a9c992df67e6d83b9a2cde42a1e available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5e0a428dec71bc493736587b73784a1f6abae0d7 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 52e6b849684f2f1766687c534061570faa63d364 available at: https://gist.github.com/b8410c1df396f7e152c7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/pr-interface-win64-PR-6685.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6685.dmg

**Pull request merged**